src/evo/static_genome.py

```diff
@@ -3788,6 +3788,58 @@
                 ]
             }
         }
+    },
+    "line_tracking": {
```

Comment:
Given line tracking is only one use-case, suggesting to change the label of this cortical area to infrared_sensor instead